src/modules/StorageProposal.rb

```diff
@@ -6095,7 +6101,14 @@ def get_proposal_vm(target, key, disk)
         )
         Builtins.y2milestone("get_proposal_vm sol:%1", disk)
       end
-      Builtins.y2milestone("get_proposal_vm ret:%1", ret)
+
+      post_processor = PostProcessor.new()
+      ret["target"] = post_processor.process_target(ret["target"])
+
+      Builtins.y2milestone(
+        "get_proposal_vm ret[ok]: %1",
+        Ops.get_boolean(ret, "ok", false)
```

Comment:
Avoid `Ops.*` methods in the new code if possible, they are just wrappers for the old YCP builtins/operators. Use the native Ruby code instead (simple `ret["ok"]`).